addons/sale/models/sale_order.py

```diff
@@ -81,6 +81,7 @@ def _rec_names_search(self):
         default='draft')
 
     client_order_ref = fields.Char(string="Customer Reference", copy=False)
+    client_order_date = fields.Char(string="Customer Reference Date", copy=False)
     create_date = fields.Datetime(  # Override of default create_date field from ORM
```

Comment:
@Feyensv Could a customer reference date make sense in generic?  (have you had this question before?)

Reply:
No, why would it be? :thinking:
This probably should go in the l10n needing this (and be confirmed by a Product Owner?)